source/features/cleanup-repo-filelist-actions.tsx

```diff
@@ -8,30 +10,34 @@ import features from '.';
 import {groupButtons} from '../github-helpers/group-buttons';
 
 function init(): void {
-	const searchButton = select('.btn[data-hotkey="t"]')!;
-	searchButton.classList.add('tooltipped', 'tooltipped-ne');
-	searchButton.setAttribute('aria-label', 'Go to file');
-	searchButton.firstChild!.replaceWith(<SearchIcon/>);
+	observe('.btn[data-hotkey="t"]', {
```

Comment:
We should keep in mind that `select` and `observe` are not equivalent. `select` will only pick one while `observe` will work as `select.all`. This could cause trouble

Reply:
The cause is `observe('get-repo details')`, this one is fine